teams/teams-ps/teams/Set-CsOnlineLisCivicAddress.md

```diff
@@ -33,6 +33,11 @@ Set-CsOnlineLisCivicAddress -CivicAddressId <Guid> [-CompanyName <String>] [-Com
 ## DESCRIPTION
 Use the `Set-CsOnlineLisCivicAddress` cmdlet to modify an existing civic address which has not been validated. Validated civic addresses cannot be modified.
 
+Address editing is restricted to the following countries/regions:
+AU, BR, CA, HR, CZ, EE, HK, HU, IL, JP, LV, LT, MX, NZ, PL, PR, RO, SG, KR, SI, ZA, US
```

Comment:
nit: "Editing address using this cmdlet is restricted to the following countries"

Also please add full name of the countries as abbreviated country name may not be very well understood.